pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1297,6 +1297,11 @@
         doc = "If true, export consumer level metrics otherwise namespace level"
     )
     private boolean exposeConsumerLevelMetricsInPrometheus = false;
+    @FieldContext(
+            category = CATEGORY_METRICS,
+            doc = "If true, export raw counter topic level metrics"
+    )
+    private boolean exposeCounterTopicMetricsInPrometheus = false;
```

Comment:
I don't think we need this flag. 

if `exposeTopicLevelMetricsInPrometheus` is true, we can expose this counter. there should be no penalty to do so.

Reply:
Agree with your point of view

Reply:
Right, doing the modifications